src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/AggregationIT.java

```diff
@@ -1116,7 +1116,7 @@ public void termsWithScript() throws Exception {
                 "'interval'='20d ', 'format'='yyyy-MM-dd') limit 1000", TEST_INDEX_ONLINE);
         String result = explainQuery(query);
 
-        Assert.assertThat(result, containsString("\"script\":{\"source\""));
+        Assert.assertThat(result, containsString("\"aggregations\" : {"));
```

Comment:
I think the json formatter only impact the output format, why the content is changed?

Reply:
Yes it only changes the format. The content keeps the same.